src/components/VolunteerApprovalGrid/Row/Row.module.css

```diff
@@ -38,8 +38,8 @@
   display: flex;
   border-radius: 5px;
   padding: 15px;
-  column-gap: 13px;
-  flex-wrap: wrap;
+  column-gap: 30px;
+  flex-wrap: nowrap;
```

Comment:
why change to nowrap here?

Reply:
column gap to give better distance between pfp section and details section. nowrap so that the details don't go to another row, just in case. I don't believe its necessary though, i didn't see it happen after I had adjusted the size of the pfp